pkg/build/scripts/grub/guestfish-snippet.tpl

```diff
@@ -0,0 +1,13 @@
+# Configure GRUB for first boot
+# - Without this, the values wouldn't be used until after the first time the
+#   grub configuration is regenerated
+download /boot/grub2/grub.cfg /tmp/grub.cfg
+! sed -i '/ignition.platform/ s/$/ {{.KernelArgs}} /' /tmp/grub.cfg
```

Comment:
I'd suggest to either add a TODO or open an issue to track a better way of customising the usage of the different args. We shouldn't be building on top of this.